modules/lvms-scaling-storage-of-clusters-using-rhacm.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-scaling-storage-of-clusters-using-rhacm_{context}"]
+= Scaling up the storage of clusters by using {rh-rhacm}
+
+You can scale up the storage capacity of the worker nodes on a cluster using {rh-rhacm}.
+
+.Prerequisites
+
+* You have access to the {rh-rhacm} cluster using an account with `cluster-admin` privileges.
+* You have created an `LVMCluster` CR by using {rh-rhacm}.
+* You have unused devices on each cluster.
+
+.Procedure
+
+. Log in to the {rh-rhacm} CLI using your {product-title} credentials.
+. Edit the `LVMCluster` CR YAML file that you created using {rh-rhacm} by running the following command:
++
+[source,terminal]
+----
+# oc edit -f <file_name> -ns <namespace> <1>
+----
+<1> Replace `<file_name>` with the name of the `LVMCluster` CR YAML file.
+
+. In the `LVMCluster` CR YAML file, add the path to the new device in the `deviceSelector` field.
```

Comment:
@jakobmoellerdev could you check if this procedure is accurate? You have more experience with RHACM.

Reply:
I suggest this procedure to be verified via QE case instead of me manually standing by that its okay. WDYT?

Reply:
@radeore do you know if we have any LVMS RHACM Test Cases?

Reply:
@jakobmoellerdev I don't think we have test cases for LVMS RHACM.

Reply:
@radeore could we add one? As this is documented in the product documentation, I think we should at least have a test case to verify if it can be installed without any problems.

Reply:
Sure, I need to first check on how to setup ACM cluster, didn't worked on it earlier as we don't have ACM in our normal testing.

@sr1kar99 I wanted to know, how did you get to know about this procedure, did anyone share you this?

Reply:
@radeore 
The following procedures related to RHACM were added in 4.12:
- Installing LVM Storage using RHACM
- Uninstalling LVM Storage using RHACM
- Scaling up storage of clusters using RHACM

Doc link for 4.12 - https://docs.openshift.com/container-platform/4.12/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.html#lvms-scaling-storage-of-single-node-openshift-cluster-using-rhacm_logical-volume-manager-storage

Reply:
RHACM procs are being tracked in this PR - https://github.com/openshift/openshift-docs/pull/73279